python/redcanary/detections.py

```diff
@@ -57,15 +72,64 @@ def all(self, per_page: int = None, since: str = None) -> list:
         """
         return [Detection(i) for i in self._get_all('detections', locals())]
 
-    def byid(self, id: int) -> object:
+    def get_by_id(self, id: int) -> object:
         """
+        Searches for detection by id
+
         Parameters
         --------
         id : int
             detection id
         """
         return Detection(self._get_by_id('detections', id))
 
+    def get_detectors(self, detection: object) -> list:
```

Comment:
```suggestion
    def detectors(self, detection: object) -> list:
```
Same as above comment